prow/pubsub/subscriber/subscriber.go

```diff
@@ -415,6 +426,10 @@ func (s *Subscriber) handleProwJob(l *logrus.Entry, jh jobHandler, msg messageIn
 		reportProwJobFailure(&prowJob, err)
 		return err
 	}
-	l.Infof("Job %q created as %q", pe.Name, prowJob.Name)
+	l.WithFields(logrus.Fields{
+		"name": pe.Name,
+		"job":  prowJob.Name,
```

Comment:
I think these are backwards.  Name should be the ProwJob resource name (a GUID) and job should be the job name defined in config.

Reply:
got it, this is confusing tbh

Reply:
Yeah I agree. It would have been clearer if `pe.Name` was `pe.Job`, but that ship has sailed. :man_shrugging: